plugins/autocomplete/plugin.js

```diff
@@ -0,0 +1,1208 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+'use strict';
+
+( function() {
+
+	// This flag prevents appending stylesheet more than once.
+	var stylesLoaded = false;
+
+	CKEDITOR.plugins.add( 'autocomplete', {
+		requires: 'textwatcher,caretposition',
+		init: function() {
+			if ( !stylesLoaded ) {
+				CKEDITOR.document.appendStyleSheet( this.path + 'skins/' + CKEDITOR.skin.name + '/autocomplete.css' );
+				stylesLoaded = true;
+			}
+		}
+	} );
+
+	/**
+	 * The main class implementing a generic {@link https://ckeditor.com/cke4/addon/autocomplete autocomplete} feature in the editor.
```

Comment:
This invalid way of creating links in documentation (Umberto throws an error for this link).
`{@link ...}` is used to link inside docs. Outside docs should be used regular markdown syntax.